src/hooks/coreSDK/safeCoreSDK.ts

```diff
@@ -19,7 +19,7 @@ export const isLegacyVersion = (safeVersion: string): boolean => {
 }
 
 export const isValidSafeVersion = (safeVersion?: SafeInfo['version']): safeVersion is SafeVersion => {
-  const SAFE_VERSIONS: SafeVersion[] = ['1.3.0', '1.2.0', '1.1.1', '1.0.0']
+  const SAFE_VERSIONS: SafeVersion[] = ['1.4.1', '1.3.0', '1.2.0', '1.1.1', '1.0.0']
```

Comment:
I would remove this altogether because ultimately it just repeats the same check in the protocol-kit. We can instead look up the version via protocol-kit or safe-deployments.